platforms/Windows/python/python.wxi

```diff
@@ -0,0 +1,159 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Include xmlns="http://wixtoolset.org/schemas/v4/wxs">
+
+  <Package
+      Language="1033"
+      Manufacturer="!(loc.ManufacturerName)"
+      Name="$(VariantProductName)"
+      UpgradeCode="$(VariantUpgradeCode)"
+      Version="$(NonSemVerProductVersion)"
+      Scope="$(PackageScope)">
+
+    <Media Id="1" Cabinet="$(VariantCabinetName)" EmbedCab="$(ArePackageCabsEmbedded)" />
+
+    <WixVariable Id="SideBySidePackageUpgradeCode" Value="$(VariantUpgradeCode)" />
+    <FeatureGroupRef Id="SideBySideUpgradeStrategy" />
+
+    <ComponentGroup Id="EmbeddedPython">
+      <Component Directory="toolchain_$(VariantName)_usr_bin">
+        <File Source="$(PythonRoot)\libcrypto-1_1-arm64.dll" />
+      </Component>
+
+      <Component Directory="toolchain_$(VariantName)_usr_bin">
+        <File Source="$(PythonRoot)\libffi-7.dll" />
+      </Component>
+
+      <Component Directory="toolchain_$(VariantName)_usr_bin">
+        <File Source="$(PythonRoot)\libssl-1_1-arm64.dll" />
+      </Component>
+
+      <Component Directory="toolchain_$(VariantName)_usr_bin">
+        <File Source="$(PythonRoot)\LICENSE.txt" />
+      </Component>
```

Comment:
Why is the `LICENSE` file in `/usr/bin`? Shouldn't that be under `/usr/share/licenses`?